src/MapboxStyleParser.ts

```diff
@@ -114,6 +114,7 @@ type SymbolType = {
 
 type OptionsType = {
   ignoreConversionErrors?: boolean;
+  replaceGraphicFillWithColor?: boolean;
```

Comment:
Could you add a small descriptive comment here, so other users get a little more context?